portal/src/helpers/state.js

```diff
@@ -48,14 +48,13 @@ export const appActions = regRedux(
       state.authenticated = payload;
     },
     setSpace(state, { error, space }) {
-      if (error) state.error = error;
-      else {
+      if (error?.statusCode === 401) {
+        state.space = {};
+      } else if (error) {
+        state.error = error;
+      } else {
         state.space = space;
-        state.kappSlug = getAttributeValue(
-          space,
-          'Service Portal Kapp Slug',
-          'service-portal',
-        );
+        state.kappSlug = getAttributeValue(space, 'Catalog', 'catalog');
```

Comment:
The second parameter to the `getAttributeValue` function is the attribute definition name. That shouldn't be changed to `Catalog` and should remain as `Service Portal Kapp Slug`. The third parameter is the default value, so it's fine to change that.